src/common/arrow/arrow_converter.cpp

```diff
@@ -124,6 +124,9 @@ void SetArrowFormat(DuckDBArrowSchemaHolder &root_holder, ArrowSchema &child, co
 	case LogicalTypeId::HUGEINT:
 		child.format = "d:38,0";
 		break;
+	case LogicalTypeId::UHUGEINT:
+		child.format = "d:38,0";
```

Comment:
One problem with this, for both UHUGEINT and HUGEINT is that their value ranges are bigger than are supported by DECIMAL(38,0)